siliconcompiler/core.py

```diff
@@ -1075,6 +1075,17 @@ def _find_files(self,
 
         is_list = bool(re.match(r'\[', paramtype))
 
+        def convert_paths_to_posix(paths):
+            posix_paths = []
+            for p in paths:
+                if p:
+                    # Cast everything to a windows path and convert to posix.
+                    # https://stackoverflow.com/questions/73682260
+                    posix_paths.append(pathlib.PureWindowsPath(p).as_posix())
+                else:
+                    posix_paths.append(p)
```

Comment:
`None`

Reply:
Maybe I'm missing something but why would we want to append that?

Reply:
Because we return a `None` for each file we cannot find (in the same order of the list).
`["found.v", "notfound.v", "found2.v"]` -> `["found.v", None, "found2.v"]`